.github/actions/confluence-release-notes/confluence_create_release_page.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+YEAR="$1" # Release year
+RELEASE_NAME="$2"
+RELEASE_END_DATE="$3"
+RELEASE_NOTES_FILE="$4"
+
+if [ "$PARENT_PAGE_ID" == "" ]; then
+    PARENT_PAGE_ID="4489150471"
+fi
+
+if [ "$UPDATE_EXISTING" == "" ]; then
+    export UPDATE_EXISTING="false" # Do not update the folder page if it exists by default
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+JIRA_CONFIG="$($SCRIPT_DIR/jira_get_config.sh)"
+ATLASSIAN_USER=$(echo $JIRA_CONFIG | jq -r .auth.user)
+ATLASSIAN_API_TOKEN=$(echo $JIRA_CONFIG | jq -r .auth.token)
+DOMAIN="$(echo $JIRA_CONFIG | jq -r .jira.domain)"
+
+if [ "$GET_PAGE_CMD" == "" ]; then
+    GET_PAGE_CMD="$SCRIPT_DIR/confluence_get_pages.sh"
+fi
+
+if [ "$CREATE_PAGE_CMD" == "" ]; then
+    CREATE_PAGE_CMD="$SCRIPT_DIR/confluence_post_page.sh"
+fi
+
+# Create titles for folder and page
+FOLDER_TITLE="$YEAR Release Notes"
+PAGE_TITLE="$RELEASE_END_DATE Release Notes - $RELEASE_NAME"
+
+show_usage() {
+    echo "Usage: [year] [release_name] [release_end_date] [release_notes_file]"
+    exit 1
+}
+
+get_parent_page() {
+    local response="$($GET_PAGE_CMD -i "$PARENT_PAGE_ID")"
+    echo $response
+}
+
+create_folder() {
```

Comment:
Same here

Reply:
Addressed in [a1901e9](https://github.com/VirdocsSoftware/github-actions/pull/20/commits/a1901e9408a0907599fa915fae895d2d9ed9120d)